docs/extensibility/resources.md

```diff
@@ -0,0 +1,226 @@
+- **advanced** - whether to diplay in the advanced form. By default it is true.
+
+### Example
+
+```json
+[
+  { "path": "spec.priority", "simple": true },
+  { "path": "spec.items.name" },
+  { "path": "spec.items.service.url" },
+  { "path": "spec.items.service.port" }
+]
+```
+
+### `list` section
+
+List section defines extra columns available in the list. The format is similar to the form section, however each entry consists only of two values:
+
+#### Item parameters
+
+- **path** - _[required]_ contains the path to the data to use for the column.
+- **widget** - optional widget used to render the field. By default the value will be displayed verbatim. For more information about the available widgets see [Display widgets](display-widgets.md).
+
+#### Example
+
+```json
+[{ "path": "spec.url" }, { "path": "spec.priority", "widget": "Badge" }]
+```
+
+### `details` section
+
+Details section defines the display structure for the details page. It contains two sections, `header` and `body`, both of which are a list of item to display in the header section and the body of the page respectively. The format of the entries is at the core similar to the form section, however it has extra options available.
+
+The resource section is required and contains basic information about the resource - namely kind and api details.
+
+#### Items parameteres
+
+- **path** - contains the path to the data to use for the widget. Not required for purely presentational widgets.
+- **name** - used for entries without `path` to define the translation source used for labels. Required if no path is present.
+- **widget** - optional widget to render the field. By default the value will be displayed verbatim. For more information about the available widgets see [Display widgets](display-widgets.md).
+- **children** - a list of child widgets to use for all `object` and `array` type fields. Not available for header widgets.
+
+Extra parameters might be available for specific widgets.
+
+#### Example
+
+```json
+{
+  "header": [
+    { "path": "metadata.name" },
+    { "path": "spec.priority", "widget": "Badge" }
+  ],
+  "body": [
+    {
+      "name": "columns",
+      "widget": "Columns",
+      "children": [
+        { "name": "left-panel", "widget": "Panel" },
+        { "name": "right-panel", "widget": "Panel" }
+      ]
+    },
+    {
+      "name": "summary",
+      "widget": "Panel",
+      "children": [
+        { "path": "metadata.name" },
+        { "path": "spec.priority", "widget": "Badge" }
+      ]
+    },
+    {
+      "path": "spec.details",
+      "widget": "CodeViewer",
+      "language": "json"
+    },
+    {
+      "path": "spec.configPatches",
+      "widget": "Panel",
+      "children": [{ "path": "applyTo" }, { "path": "match.context" }]
+    },
+    {
+      "path": "spec.configPatches",
+      "widget": "Table",
+      "columns": [{ "path": "applyTo" }, { "path": "match.context" }]
```

Comment:
Using the `Table` widget, the `columns` key doesn't work for me. I changed `columns` to `children` and it started working. 
Using `columns` causes the error below.
![image](https://user-images.githubusercontent.com/56163767/172806653-a4c8b2e7-5ebe-4b7a-9e6f-972ee835ed7a.png)

Using `children` displays the Table widget properly.
![image](https://user-images.githubusercontent.com/56163767/172806387-f448e68d-9cf5-4e6f-8c83-02be491328ef.png)

Reply:
```suggestion
      "widget": "Table",
      "children": [{ "path": "applyTo" }, { "path": "match.context" }]
```